Makefile

```diff
@@ -80,14 +80,14 @@ install-lib: install-mod Makefile src/core/version.h
 RELEASE_FILES=$(shell echo src/core/version.h; git ls-files --recurse-submodules)
 ifeq ($(VKDT_USE_RAWINPUT), 1)
   RAWSPEED_DIR=$(shell ls -d src/pipe/modules/i-raw/rawspeed-*)
-  RELEASE_FILES+=$(shell cd $(RAWSPEED_DIR) && git ls-files | sed -e 's#^#$(RAWSPEED_DIR)/#')
```

Comment:
did this actually lead to problems on your system? i would say these # pounds need no escaping because they are within ' single quotes, and this works as it is on my end.

Reply:
Yes, I have this error on branch `master`:

```
$ make clean
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C .. clean
Makefile:48: [ ()]
Makefile:92: [ ()]
Makefile:87: *** unterminated call to function `shell': missing `)'.  Stop.
make: *** [clean] Error 2
```

Line 87:
```
  RELEASE_FILES+=$(shell cd $(MCRAW_DIR) && git ls-files | sed -e 's#^#$(MCRAW_DIR)/#')
```

Reply:
which version of make is this? this even worked on my tests with macos.

Reply:
the native version that comes with Mac is 3.81, installing a newer version with brew worked for me.